Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodeGraphWidget.cpp

```diff
@@ -1095,7 +1087,7 @@ namespace EMStudio
                 AZ_Assert(!mActiveGraph->IsInReferencedGraph(), "Expected to not be in a referenced graph");
 
                 // get the information from the current mouse position
-                uint32      newTargetPortNr;
+                AZ::u16     newTargetPortNr;
```

Comment:
Why AZ::u16 here? Can we align to a standard with this change now that we're changing it anyway?

Reply:
That variable is passed to `NodeGraph::FindPort`, which calls `GraphNode::FindPort(int32 x, int32 y, AZ::u16* outPortNr, bool* outIsInputPort, bool includeInputPorts)`. The signature of that method was changed to match the type that that function uses internally. That snippet is this:
```cpp
  1003 const AZ::u16 numInputPorts = aznumeric_caster(mInputPorts.size());
  1004 for (AZ::u16 i = 0; i < numInputPorts; ++i)
  1005 {
  1006     QRect rect = CalcInputPortRect(i);
  1007     if (rect.contains(QPoint(x, y)))
  1008     {
  1009         *outPortNr      = i;
  1010         *outIsInputPort = true;
  1011         return &mInputPorts[i];
  1012     }
  1013 }
```

It was already using `AZ::u16`, so I just changed the parameter type to match.

Reply:
If we want to do a separate pass to unify our `uint32` and `AZ::u32` and `uint32_t`, I'm all for it, but I think that's separate from this change.

Reply:
Sounds good